docs/userGuide/syntax/lists.md

```diff
@@ -109,3 +109,97 @@ first number
 - [x] Item 4
 - ( ) Item 5
 </div>
+
+****Customizing the list appearance:****
+
+**To customize list icons, add the configuration `{icon="icon-name"}` either after a specific list item or at the end of the entire list.**
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">markdown</variable>
+<variable name="code">
+* item 1 {icon="fas fa-times"}
```

Comment:
> To customize list icons, add the configuration `{icon="icon-name"}` either after a specific list item or at the end of the entire list.

👀 Are we not aiming to maintain the existing icon syntax per this and your prior discussion? (without the `::`'s)

https://deploy-preview-2316--markbind-master.netlify.app/userguide/formattingcontents#icons

e.g.

- `fas fa-times` -> `fas-fa-times`
- `glyphicon glyphicon-education` -> `glyphicon-education`

p.s: I actually think removing the `::`'s here is a more intuitive option as its a delimiter used in markdown text, not as an attribute.

Reply:
https://github.com/MarkBind/markbind/blob/master/packages/core/src/lib/markdown-it/plugins/markdown-it-icons.ts

Reply:
@SPWwj as @ang-zeyu suggested, let's use the same icon syntax, minus the `:` delimiter.

Reply:
> p.s: I actually think removing the `::`'s here is a more intuitive option as its a delimiter used in markdown text, not as an attribute.

@ang-zeyu Makes sense. That said, we already use `icon=":fas-home:"` syntax for boxes.

Reply:
'fas fa-times' is a Bootstrap class, not an emoji. For users familiar with Bootstrap, it makes more sense to retain the original icon syntax 'fas fa-times' rather than converting it to a new syntax like 'fas-fa-times'.

The same goes for the emoji 'glyphicon glyphicon-education'.

Since these are all included in the official icon list, I believe there's no need for us to alter them.

Reply:
@SPWwj we want to be as consistent as possible within MarkBind, before we try be consistent with Bootstrap. Icons are already an existing feature, and we should be able to say  "use the same syntax as [the icons feature]()" or failing that, "use the same syntax as [the icons feature]() but omit `:`"

Reply:
> @SPWwj we want to be as consistent as possible within MarkBind, before we try be consistent with Bootstrap. Icons are already an existing feature, and we should be able to say "use the same syntax as the icons feature" or failing that, "use the same syntax as the icons feature but omit `:`"

I will try to edit acordingly.